test/parallel/sdk.API.utils.test.ts

```diff
@@ -170,16 +170,12 @@ test("GIVEN multiSwapBuy WHEN operation is confirmed AND isMultiSwapAssetTransac
 });
 
 test("GIVEN multiSwapBuy WHEN operation is failed AND isMultiSwapAssetTransactionSuccessful THEN it returns false", async () => {
-  const [token2] = await Assets.setupUserWithCurrencies(
-    sudo,
-    [defaultCurrencyValue],
-    sudo,
-  );
-
-  const tokenIds = [token1, token2];
+  const [testUser2] = setupUsers();
+  await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser2));
+  const tokenIds = [token1, GASP_ASSET_ID];
```

Comment:
why?

Reply:
I found another way. We can use BN for a non-existent liquid that will not exist in our tests for a long time. E.g., BN_MILLION.